app/grants/templates/grants/components/collection.html

```diff
@@ -33,9 +33,10 @@
               <img :src="getGrantLogo(3)" alt="" v-if="collection.grants.length >= 4">
             </div>
           </div>
-          <div class="collection-item__owner-image--cover">
+          <div class="collection-item__owner-image--cover" v-if="!collection.cover">
             <img :src="collection.owner.avatar_url || 'https://c.gitcoin.co/avatars/57e79c0ae763bb27095f6b265a1a8bf3/thelostone-mc.svg'">
```

Comment:
wait this is from the old code but if not user is default to thelostone avatar? users always have an avatar_url, if not the fallback should be github one, the same way we do in other places